controller/work_item_events.go

```diff
@@ -134,8 +124,8 @@ func ConvertEvent(appl application.Application, request *http.Request, wiEvent e
 			},
 		}
 	case workitem.SystemIteration:
-		old := ConvertIterationSimple(request, wiEvent.Old)
-		new := ConvertIterationSimple(request, wiEvent.New)
+		old, _ := ConvertIterationSimple(request, wiEvent.Old)
+		new, _ := ConvertIterationSimple(request, wiEvent.New)
```

Comment:
Why can't we add the links in the relationships?

Reply:
see https://github.com/fabric8-services/fabric8-wit/pull/2205#discussion_r207145107